Source/WebCore/style/StyleBuilderState.cpp

```diff
@@ -172,7 +172,7 @@ bool BuilderState::createFilterOperations(const CSSValue& inValue, FilterOperati
                 continue;
 
             auto filterURL = primitiveValue.stringValue();
-            auto fragment = document().completeURL(filterURL).fragmentIdentifier().toString();
+            auto fragment = document().completeURL(filterURL).fragmentIdentifier().toAtomString();
             operations.operations().append(ReferenceFilterOperation::create(filterURL, fragment));
```

Comment:
No `WTFMove`?

Reply:
Yes, looks like I can move it all the way to the ReferenceFilterOperation data member, nice.